embassy-stm32/src/flash/mod.rs

```diff
@@ -65,9 +65,11 @@ impl FlashRegion {
 #[cfg_attr(flash_f7, path = "f7.rs")]
 #[cfg_attr(flash_g0, path = "g0.rs")]
 #[cfg_attr(flash_h7, path = "h7.rs")]
+#[cfg_attr(flash_h7ab, path = "h7ab.rs")]
```

Comment:
since both are so similar, it's nicer to have a single `h7.rs` file handling both. Also notice both versions have CRCRDERR, I snuck in the fix for that in your stm32-data PR.

Reply:
good point